manifests/overlays/prod/configs/argo_cm/resource.inclusions

```diff
@@ -435,3 +487,46 @@
   - AirflowCluster
   clusters:
   - https://paas.stage.psi.redhat.com:443
+- apiGroups:
+  - argoproj.io
+  kinds:
+  - CronWorkflow
+  - EventBus
+  - EventSource
+  - Gateway
+  - Sensor
+  - Workflow
+  - WorkflowTemplate
+  clusters:
+  - https://paas.stage.psi.redhat.com:443
+- apiGroups:
+  - argoproj.io
+  kinds:
+  - CronWorkflow
+  - Workflow
+  - WorkflowTemplate
+  clusters:
+  - https://k-openshift.osh.massopen.cloud:8443/
+- apiGroups:
+  - operators.coreos.com
+  kinds:
+  - CatalogSource
+  - ClusterServiceVersion
+  - InstallPlan
+  - Subscription
+  clusters:
+  - https://k-openshift.osh.massopen.cloud:8443/
+- apiGroups:
+  - tekton.dev
+  kinds:
+  - Condition
+  - EventListener
+  - PipelineResource
+  - PipelineRun
+  - Pipeline
+  - TaskRun
+  - Task
+  - TriggerBinding
+  - TriggerTemplate
```

Comment:
TriggerBinding, TriggerTemplate and EventListener are of different apiGroups: trigger.tekton.dev i think.
also can we lose , pipelinerun and taskrun in here.

Reply:
ok will remove pipelinerun/taskrun, but for the apigroup the output I have from you from: 
`kubectl api-resources --namespaced=true` shows the apigroup as `tekton.dev`

Reply:
updated to be `trigger.tekton.dev`